strawberryfields/apps/qchem/utils.py

```diff
@@ -96,3 +96,59 @@ def duschinsky(
     delta = np.array(d @ l0_inv)
 
     return U, delta
+
+
+def read_gamess(file) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    r"""Reads molecular data from the output file generated by the GAMESS quantum chemistry package.
+
+    This function extracts the atomic coordinates, atomic masses, vibrational frequencies,
+    and normal modes of a molecule from the output file of a vibrational frequency calculation
+    performed with the GAMESS quantum chemistry package.
+
+    **Example usage:**
+
+    >>> r, m, w, l = read_gamess('../BH_frq.out')
+    >>> r
+    array([0.000000, 0.000000, 0.000000, 1.253604, 0.000000, 0.000000]
+
+    Args:
+        file: name and the path of the GAMESS output file
+
+    Returns:
+        tuple[array, array, array, array]: atomic coordinates, atomic masses, normal mode
+        frequencies, normal modes
+    """
+    with open(file, "r") as f:
```

Comment:
One general comment: How confident are we that GAMESS files always "look the same"? This function is relying heavily on these files having a very specific, reliable structure. Is this a good assumption?

Reply:
For example, I wondered if we should raise any errors. But raising errors would mean there is a problem with the file, right? 
Maybe we should at least test that the input is a GAMESS file

Reply:
To my knowledge the format is pretty standard for vibrational frequency calculations. I tested the function on two outputs generated independently by myself and Alain for different molecules. I will add one or few conditions for raising errors.

Reply:
@soranjh, probably to explore: GAMESS outputs a file with extension ".dat" which I think it is precisely intended for post-processing purposes. That's why the format of this file is purely numeric. I think it contains the information you are looking for (after the molecular orbitals printing group). Perhaps, it might make the function simpler(not 100% sure). Just attached a sample of this file below. (just hacked the extension of the file to be ".log" as github does not support ".dat"
[dft_hessian_gs_b3lyp-ccpVTZ.log](https://github.com/XanaduAI/strawberryfields/files/5285160/dft_hessian_gs_b3lyp-ccpVTZ.log)